### ISSUE TITLE
Not working!!

### ISSUE THREAD

**Issue by sanihaq** [willasm/project-notes#2]
Not working after installing on my machine. Tried disabling all extension and  running only this but still did not work. 
Your project looks interesting, please keep going. Good luck.👍

**Comment by willasm** [comment]
Can you describe what you are seeing. Is there any error messages you are getting? Does the project notes icon appear in the side bar? Does the todo syntax highlighting not appear for you? Please give me as many details as possible so I can correct any issues, thank you, William.

**Comment by sanihaq** [comment]
- all `create new` buttons not working. 
- getting this error when using any command
![2022-07-24 19_49_23-Window](https://user-images.githubusercontent.com/51208318/180651305-256e7edf-a86e-47a2-bf5e-bbaf2c86cba2.png)

**Comment by willasm** [comment]
I just did a clean install of vscode with no extensions installed then added my extension and I can confirm that I received the exact same error message. It does work as expected if I have a code file opened then reload the window or restart vscode. Try running from the command pallet "Developer: Reload Window" with any file opened to confirm this.

After testing this for a while I can confirm that the extension is failing to load if vscode is opened without a file loaded, but it does work if vscode is opened with a file loaded (Can be any type of file). I'm a little puzzled why this is happening? I do have a couple of ideas I want to try though.

Thanks for bringing this to my attention. I always open vscode with the last file I was working on so I never got this error...
Will let you know when I have found the problem.. William
